CMakeLists.txt

```diff
@@ -233,6 +233,29 @@ else()
     add_library(ADLMIDI ALIAS ADLMIDI_static)
 endif()
 
+if(WITH_MIDIPLAY OR WITH_ADLMIDI2)
+    find_library(SDL2_LIBRARY SDL2 REQUIRED)
+    message("Found ${SDL2_LIBRARY}")
```

Comment:
I'd rather see this as whole code as a find module. You could use ours: https://github.com/Return-To-The-Roots/s25client/blob/master/cmake/Modules/FindSDL2.cmake